services/networking/vpc/controls.yaml

```diff
@@ -26,24 +26,19 @@ controls:
     test_requirements:
       - id: CCC.VPC.C01.TR01
         text: |
-          Verify that default networks are not automatically created upon project initialization.
-        tlp_levels:
-          - tlp_red
-      - id: CCC.VPC.C01.TR02
-        text: |
-          Confirm that only custom networks with appropriate security controls are in place.
+          When a subscription is created, the subscription must not contain any
+          default network resources.
         tlp_levels:
           - tlp_red
 
   - id: CCC.VPC.C02
-    title: Limit External IP Addresses for Virtual Machines
+    title: Limit External IP Addresses
     objective: |
-      Restrict the assignment of external (public) IP addresses to virtual machines to reduce
-      exposure to the public internet and minimize attack surfaces.
+      Restrict the assignment of external (public) IP addresses to resources to
```

Comment:
Should this be reworded as "Restrict creation of resources and services in the public subnet with direct access to internet, minimizing the attack radius."